### PR TITLE
feat: New spec and parser to get capsules and repos with contidions

### PR DESCRIPTION
**Pull request by huali027**
* Also reorder the classes in satellite_postgresql_query by alphabetically.

Signed-off-by: Huanhuan Li <huali@redhat.com>

### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### CODE REVIEW

**Comment by huali027** [comment]
@xiangce Please help to review again, thanks.

File: insights/parsers/satellite_postgresql_query.py [hunk 1]
```diff
@@ -182,9 +186,32 @@ class SatelliteComputeResources(SatellitePostgreSQLQuery):
...
+        >>> tasks[0]['count']
+        '0'
+    """
+    columns = ['count']
+
+
 @parser(Specs.satellite_katello_empty_url_repositories)
 class SatelliteKatelloEmptyURLRepositories(SatellitePostgreSQLQuery):
     """
+    .. warning::
```
Comment (psachin) [inline insights/parsers/satellite_postgresql_query.py:211]: Please use https://insights-core.readthedocs.io/en/latest/api.html?highlight=deprecate#feature-deprecation

Comment (huali027) [reply insights/parsers/satellite_postgresql_query.py:212]: @psachin, thanks for the review. I update it, please help to review again.

**Review by psachin — APPROVED** [review]

File: insights/parsers/satellite_postgresql_query.py [hunk 2]
```diff
@@ -8,12 +8,16 @@
...
 SatelliteComputeResources - command ``psql -d foreman -c 'select name, type from compute_resources' --csv``
 -----------------------------------------------------------------------------------------------------------
-SatelliteSCAStatus - command ``psql -d candlepin -c "select displayname, content_access_mode from cp_owner" --csv``
--------------------------------------------------------------------------------------------------------------------
-SatelliteKatelloEmptyURLRepositories - command ``psql -d foreman -c 'select id, name from katello_root_repositories where url is NULL;' --csv``
------------------------------------------------------------------------------------------------------------------------------------------------
 SatelliteCoreTaskReservedResourceCount - command ``psql -d pulpcore -c 'select count(*) from core_taskreservedresource' --csv``
 -------------------------------------------------------------------------------------------------------------------------------
+SatelliteKatelloEmptyURLRepositories - command ``psql -d foreman -c 'select id, name from katello_root_repositories where url is NULL;' --csv``
```
Comment (xiangce) [inline insights/parsers/satellite_postgresql_query.py:13]: If this parser is going to be deprecated, it would be better to remove it from the list.

Comment (huali027) [reply insights/parsers/satellite_postgresql_query.py]: @xiangce updated, please help to review again, thanks.

**Review by xiangce — APPROVED** [review]

**Pull request merged**